debian/postinst

```diff
@@ -15,6 +15,9 @@
 # limitations under the License.
 #
 
+DEFAULT_UDEV_NET_SETUP_LINK_DIR="/usr/lib/udev/rules.d"
+MODIFIED_UDEV_NET_SETUP_LINK_DIR="/etc/udev/rules.d"
```

Comment:
I don't feel strongly, but IMO, I'd define these closer to the logic that'll actually use them..

Reply:
yes, I have moved the initialization closer to the actual logic.